extensions/jta/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,13 @@
+This extension could be used when you need the `transacted()` in the router. It leverages the transaction capability provided by the narayana-jta extension in the Quarkus. 
+
+
+Refer to the https://quarkus.io/guides/transaction[Quarkus Transaction guide] for the more details about the transaction. For a simple usage:
```

Comment:
```suggestion
This extension should be added when you need to use the `transacted()` EIP in the router. It leverages the transaction capabilities provided by the narayana-jta extension in Quarkus. 


Refer to the https://quarkus.io/guides/transaction[Quarkus Transaction guide] for the more details about transaction support. For a simple usage:
```

Reply:
Thanks @jamesnetherton - it looks good to me !